src/basilisp/core.lpy

```diff
@@ -4029,11 +4029,11 @@
   "Time the execution of ``expr``\\. Return the result of ``expr`` and print the time
   execution took in milliseconds."
   [expr]
-  `(let [start (perf-counter)]
+  `(let [start (#'perf-counter)]
```

Comment:
Can we just make `perf-counter` public with a docstring thar says it's an implementation detail?

Reply:
Done